tslint.json

```diff
@@ -1,5 +1,8 @@
 {
-    "extends": ["tslint:latest"],
+    "extends": [
+        "tslint:latest",
+        "tslint-config-prettier"
```

Comment:
not compatible with hound at this point. see https://github.com/houndci/hound/issues/1526